app/routes.py

```diff
@@ -1 +1,278 @@
-from flask import Blueprint
\ No newline at end of file
+'''
+ROUTES
+-------------------------
+Defines endpoints for *tasks* and *goals* for the Task List API.
+
+TABLE OF CONTENTS:
+    [0] Imports
+    [1] Helper Functions
+    [2] Blueprints
+    [3] *TASK* endpoints
+    [4] *GOAL* endpoints
+'''
+
+
+
+##### IMPORTS #################################################################
+
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
+import datetime
+import requests
+import os
+
+
+
+
+##### HELPER FUNCTIONS ########################################################
+
+def validate_object(object_id, object_type):
+    '''
+    Validates the goal or task based on ID and fetches the object from the database.
+        *object_id:  id of a task or goal
+        *object_type: "goal" or "task" depending on endpoint
+        OUTPUT: goal or task object fetched from database.
+    '''
+    try:
+        object_id = int(object_id)
+    except:
+        abort(make_response({"message":f"Object {object_id} invalid"}, 400))
+
+    if object_type == "task":
+        goal_or_task = Task.query.get(object_id)
+    elif object_type == "goal":
+        goal_or_task = Goal.query.get(object_id)
+    if not goal_or_task:
+        abort(make_response({"message":f"Object {object_id} not found"}, 404))
+
+    return goal_or_task
```

Comment:
If you pass in the class itself (instead of a string) you can make this more generic:

```suggestion
    model_obj = object_type.query.get(object_id)
    
    if not model_obj:
        abort(make_response({"message":f"Object {object_id} not found"}, 404))

    return model_obj
```